packages/armadillo.rb

```diff
@@ -8,17 +8,21 @@ class Armadillo < Package
   source_url 'https://downloads.sourceforge.net/project/arma/armadillo-8.400.0.tar.xz'
   source_sha256 '5cb6bc2f457a9d6a0758cfb15c418d48289909daccd79d0e428452029285dd9b'
 
+  depends_on 'cmake' => :build
   depends_on 'openblas'
 
   def self.build
     system "./configure",
```

Comment:
Should this line be `system "cmake",` instead?

Reply:
Their `configure` script is just a wrapper around `cmake`.  These arguments get forwared on to `cmake`.  I left it as using `configure` because that's what they say in their documentation.

Reply:
But cmake is not executed, only make?

Reply:
`cmake` is executed.  The `configure` bascially just forwards all arguments to `cmake`.  The final line of the `configure` script is `cmake $@ .`.

Reply:
OK, thanks for the clarification.  It makes sense now.

Reply:
Sure thing.  👍